guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,159 @@
+[id="Upgrading-Project-from-EL7-to-EL8{context}"]
+= Upgrading {Project} from {RHEL} 7 to {RHEL} 8
+
+Use this procedure to upgrade your {Project} installation from {RHEL} 7 to {RHEL} 8.
+LEAPP is the supported tool for {RHEL} 7 to {RHEL} 8 upgrades.
+
+.Prerequisites
+* {Project} {ProjectVersionPrevious} running on CentOS 7.
+- Upgrading does not work on {RHEL}. You must have a {RHEL}-compatible LEAPP build to upgrade.
+* Access to a local repository server mirroring all required repositories.
+* No outstanding updates.
+* You can upgrade a CentOS7 system before you upgrade {Project}.
+
+.Procedure
+. Configure the https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/[@theforeman/leapp COPR Repository], which contains newer LEAPP packages than those shipped by https://wiki.almalinux.org/elevate/[AlmaLinux/ELevate], and support {Project} upgrades:
++
+----
+# curl -o /etc/yum.repos.d/theforeman-leapp.repo https://copr.fedorainfracloud.org/coprs/g/theforeman/leapp/repo/epel-7/group_theforeman-leapp-epel-7.repo
+----
+. Install required packages:
++
+----
+# yum install leapp leapp-repository leapp-data-centos
+----
+
+. Add {Project} specific repositories to `/etc/leapp/files/leapp_upgrade_repositories.repo`:
```

Comment:
please have a look at https://github.com/theforeman/theforeman.org/pull/1977, I've updated this section with the "client" repo, which should be reflected here